Index: .idea/sonarlint/securityhotspotstore/index.pb
===================================================================
diff --git a/.idea/sonarlint/securityhotspotstore/index.pb b/.idea/sonarlint/securityhotspotstore/index.pb
--- a/.idea/sonarlint/securityhotspotstore/index.pb	
+++ b/.idea/sonarlint/securityhotspotstore/index.pb	
@@ -1,0 +1,15 @@
+
+g
+7client/src/app/pages/dashboard/dashboard.component.html,6\7\673750d39a9a297592839cb0fda9775a8314c54a
+i
+9client/src/app/components/side-bar/side-bar.component.css,a\4\a4dd56664eddc26de3268a201e2bb212d15bed92
+j
+:client/src/app/components/side-bar/side-bar.component.html,0\4\043fcd974e7ffb16349f4ef8bd1c1864b0517fc8
+Q
+!client/src/app/app.component.html,d\f\dfd11a06262e96f3cd670adda25464ac3c49526b
+h
+8client/src/app/components/side-bar/side-bar.component.ts,5\f\5f011985a6ffafbef6d7556c20884d8f5e916b7a
+O
+client/src/app/app.component.ts,3\b\3bdb36930ddd309b0ed730ab181ebce164c1164b
+]
+-client/src/app/pages/home/home.component.html,2\2\22472af7b80f80dd437d099ba8bd5afff56623ee
\ No newline at end of file
Index: .idea/sonarlint/issuestore/index.pb
===================================================================
diff --git a/.idea/sonarlint/issuestore/index.pb b/.idea/sonarlint/issuestore/index.pb
--- a/.idea/sonarlint/issuestore/index.pb	
+++ b/.idea/sonarlint/issuestore/index.pb	
@@ -1,0 +1,15 @@
+
+g
+7client/src/app/pages/dashboard/dashboard.component.html,6\7\673750d39a9a297592839cb0fda9775a8314c54a
+i
+9client/src/app/components/side-bar/side-bar.component.css,a\4\a4dd56664eddc26de3268a201e2bb212d15bed92
+j
+:client/src/app/components/side-bar/side-bar.component.html,0\4\043fcd974e7ffb16349f4ef8bd1c1864b0517fc8
+Q
+!client/src/app/app.component.html,d\f\dfd11a06262e96f3cd670adda25464ac3c49526b
+h
+8client/src/app/components/side-bar/side-bar.component.ts,5\f\5f011985a6ffafbef6d7556c20884d8f5e916b7a
+O
+client/src/app/app.component.ts,3\b\3bdb36930ddd309b0ed730ab181ebce164c1164b
+]
+-client/src/app/pages/home/home.component.html,2\2\22472af7b80f80dd437d099ba8bd5afff56623ee
\ No newline at end of file
Index: e-learning.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module version=\"4\">\r\n  <component name=\"SonarLintModuleSettings\">\r\n    <option name=\"uniqueId\" value=\"69c9853f-c64a-4625-b45b-3c45ad0de244\" />\r\n  </component>\r\n</module>
===================================================================
diff --git a/e-learning.iml b/e-learning.iml
--- a/e-learning.iml	
+++ b/e-learning.iml	
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module version="4">
   <component name="SonarLintModuleSettings">
-    <option name="uniqueId" value="69c9853f-c64a-4625-b45b-3c45ad0de244" />
+    <option name="uniqueId" value="5b75d557-8531-4311-923c-7fc1142e349f" />
   </component>
 </module>
\ No newline at end of file
