Index: .idea/sonarlint/securityhotspotstore/index.pb
===================================================================
diff --git a/.idea/sonarlint/securityhotspotstore/index.pb b/.idea/sonarlint/securityhotspotstore/index.pb
--- a/.idea/sonarlint/securityhotspotstore/index.pb	
+++ b/.idea/sonarlint/securityhotspotstore/index.pb	
@@ -1,0 +1,13 @@
+
+g
+7client/src/app/pages/dashboard/dashboard.component.html,6\7\673750d39a9a297592839cb0fda9775a8314c54a
+]
+-client/src/app/pages/auth/auth.component.html,2\9\2938a3234253c3053cec47ab1aee86f08bdc03dc
+j
+:client/src/app/components/side-bar/side-bar.component.html,0\4\043fcd974e7ffb16349f4ef8bd1c1864b0517fc8
+h
+8client/src/app/components/side-bar/side-bar.component.ts,5\f\5f011985a6ffafbef6d7556c20884d8f5e916b7a
+O
+client/src/app/app.component.ts,3\b\3bdb36930ddd309b0ed730ab181ebce164c1164b
+]
+-client/src/app/pages/home/home.component.html,2\2\22472af7b80f80dd437d099ba8bd5afff56623ee
\ No newline at end of file
Index: .idea/sonarlint/issuestore/index.pb
===================================================================
diff --git a/.idea/sonarlint/issuestore/index.pb b/.idea/sonarlint/issuestore/index.pb
--- a/.idea/sonarlint/issuestore/index.pb	
+++ b/.idea/sonarlint/issuestore/index.pb	
@@ -1,0 +1,13 @@
+
+g
+7client/src/app/pages/dashboard/dashboard.component.html,6\7\673750d39a9a297592839cb0fda9775a8314c54a
+]
+-client/src/app/pages/auth/auth.component.html,2\9\2938a3234253c3053cec47ab1aee86f08bdc03dc
+j
+:client/src/app/components/side-bar/side-bar.component.html,0\4\043fcd974e7ffb16349f4ef8bd1c1864b0517fc8
+h
+8client/src/app/components/side-bar/side-bar.component.ts,5\f\5f011985a6ffafbef6d7556c20884d8f5e916b7a
+O
+client/src/app/app.component.ts,3\b\3bdb36930ddd309b0ed730ab181ebce164c1164b
+]
+-client/src/app/pages/home/home.component.html,2\2\22472af7b80f80dd437d099ba8bd5afff56623ee
\ No newline at end of file
Index: .idea/e-learning.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"WEB_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\">\r\n    <content url=\"file://$MODULE_DIR$\">\r\n      <excludeFolder url=\"file://$MODULE_DIR$/.tmp\" />\r\n      <excludeFolder url=\"file://$MODULE_DIR$/temp\" />\r\n      <excludeFolder url=\"file://$MODULE_DIR$/tmp\" />\r\n    </content>\r\n    <orderEntry type=\"inheritedJdk\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n</module>
===================================================================
diff --git a/.idea/e-learning.iml b/.idea/e-learning.iml
--- a/.idea/e-learning.iml	
+++ b/.idea/e-learning.iml	
@@ -9,4 +9,7 @@
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
   </component>
+  <component name="SonarLintModuleSettings">
+    <option name="uniqueId" value="9ca06440-7573-4da8-86a5-0cabaf92c0df" />
+  </component>
 </module>
\ No newline at end of file
